src/index.css

```diff
@@ -1,13 +1,17 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
+  font-family: "Spartan";
```

Comment:
Those 4 fonts from line 6 to 9 are some fonts I get from Google Font. Some of them have been used and others are not because the old one not looks so good. 
If you don't like them, we can delete.
At line 7 of `public\index.html` you can see how and where they were related.